src/main/java/core/selenium/WebDriverManager.java

```diff
@@ -63,6 +66,7 @@ public WebDriver getDriver() {
 
     /**
      * Method to let get a wait.
```

Comment:
write again your documentation, must be written in third person

Reply:
That is write in third person.